src/wandbot/models/llm.py

```
@@ -249,13 +249,85 @@ async def create(self,
         super().__init__(**kwargs)
         api_key = os.getenv("GOOGLE_API_KEY")
         if not api_key:
             raise ValueError("GOOGLE_API_KEY environment variable not set")
         self.client = genai.Client(api_key=api_key)
+        self.thinking_level = kwargs.get("thinking_level")
         self.thinking_budget = kwargs.get("thinking_budget")
+        if "thinking_level" not in genai_types.ThinkingConfig.model_fields:
+            raise ValueError(
+                "google-genai SDK is too old for Gemini 3 thinking_level support. "
+                "Upgrade to google-genai>=1.64.0."
+            )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)